app/views/spree/admin/taxons/_form.html.erb

```diff
@@ -0,0 +1,56 @@
+<div data-hook="admin_inside_taxon_form">
+  <%= f.field_container :name, class: ['form-group'] do %>
+    <%= f.label :name, raw(Spree.t(:name) + required_span_tag) %>
+    <%= text_field :taxon, :name, class: 'form-control' %>
+    <%= f.error_message_on :name, class: 'error-message' %>
+  <% end %>
+  <%= f.field_container :permalink, class: ['form-group'] do %>
+    <%= label_tag :permalink_part, raw(Spree.t(:permalink) + required_span_tag) %>
+    <div class="input-group mb-3">
+      <div class="input-group-prepend">
+        <span class="input-group-text" id="basic-addon3">
+          <%= [current_store.formatted_url, 't', @parent_permalink].join('/') %>
+        </span>
+      </div>
+      <%= text_field_tag :permalink_part, @permalink_part, class: 'form-control', required: true %>
+    </div>
+  <% end %>
+  <%= f.field_container :hide_from_nav, class: ['form-group'] do %>
+    <%= f.label :hide_from_nav do %>
+      <%= Spree.t(:hide_from_subcategories_nav) %>
+      <br>
+      <%= f.check_box :hide_from_nav, { checked: @taxon.hide_from_nav } %>
+      <small><%= Spree.t(:say_yes) %></small>
+    <% end %>
+  <% end %>
+  <%= f.field_container :icon, class: ['form-group'] do %>
+    <%= f.label :icon, Spree.t(:header_banner) %>
+    <%= image_tag(main_app.url_for(@taxon.icon.try(:attachment)), class: 'w-100') if @taxon.icon %>
+    <%= f.file_field :icon %>
+    <% if @taxon.icon.present? %>
+      <%= link_to Spree.t(:remove_image),
+        remove_icon_admin_taxonomy_taxon_url(@taxonomy.id, @taxon.id),
+        method: :delete %>
+    <% end %>
+  <% end %>
+  <%= f.field_container :description, class: ['form-group'] do %>
+    <%= f.label :description, Spree.t(:description) %>
+    <%= f.text_area :description, class: "form-control #{'spree-rte' if taxon_wysiwyg_editor_enabled? }", rows: 20 %>
+  <% end %>
+  <%= f.field_container :meta_title, class: ['form-group'] do %>
+    <%= f.label :meta_title, Spree.t(:meta_title) %>
+    <%= f.text_field :meta_title, class: 'form-control', rows: 6 %>
+  <% end %>
+  <%= f.field_container :meta_description, class: ['form-group'] do %>
+    <%= f.label :meta_description, Spree.t(:meta_description) %>
+    <%= f.text_area :meta_description, class: 'form-control', rows: 6 %>
+  <% end %>
+  <%= f.field_container :meta_keywords, class: ['form-group'] do %>
+    <%= f.label :meta_keywords, Spree.t(:meta_keywords) %>
+    <%= f.text_field :meta_keywords, class: 'form-control', rows: 6 %>
+  <% end %>
+  <%= f.field_container :users, class: ['form-group'] do %>
```

Comment:
Please write. override to add users dropdown.

Reply:
@Umer, Please handle this point yourself or guide hamza